tests/test_routes.py

```diff
@@ -43,3 +47,17 @@ def test_index(self):
         """ It should call the home page """
         resp = self.client.get("/")
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
+
+    def test_delete_customer(self):
```

Comment:
must test the response codes as well, HTTP_204_NO_CONTENT and HTTP_404_NOT_FOUND

Reply:
added corresponding tests and keep them consistent with the examples